rblib/format.rb

```diff
@@ -60,8 +60,8 @@ def self.make_clickable(text, params = {})
             # http://www.whatdotheyknow.com/request/advice_sought_from_information_c#incoming-24711
             ret = ret.gsub(/( LTCODE http[\S\r\n]*? GTCODE )/) { |m| m.gsub(/[\n\r]/, "") }
 
-            ret = ret.gsub(/(https?):\/\/([^\s<>{}()]+[^\s.,<>{}()])/i, "<a href='\\1://\\2'" + (nofollow ? " rel='nofollow'" : "") + ">\\1://\\2</a>")
-            ret = ret.gsub(/(\s)www\.([a-z0-9\-]+)((?:\.[a-z0-9\-\~]+)+)((?:\/[^ <>{}()\n\r]*[^., <>{}()\n\r])?)/i,
+            ret = ret.gsub(/(https?):\/\/([^\s<>]+[^\s.,<>])/i, "<a href='\\1://\\2'" + (nofollow ? " rel='nofollow'" : "") + ">\\1://\\2</a>")
+            ret = ret.gsub(/(\s)www\.([a-z0-9\-]+)((?:\.[a-z0-9\-\~]+)+)((?:\/[^ <>\n\r]*[^., <>\n\r])?)/i,
```

Comment:
Given comma and dot are allowed at the end of URLs though we ignore them here in case they're being used in a sentence, should we perhaps maintain no `)` at the end too, in case the URL is in brackets? Not sure how many URLs would have bracket right at the end as opposed to in the middle (same goes for comma/dot I guess).